README.md

```diff
@@ -47,7 +47,8 @@ Deploy now to [Gatsby Cloud](https://gatsbyjs.com/products/cloud):
 
    - Install the [Contentful CLI (installation instructions)](https://www.contentful.com/developers/docs/tutorials/cli/installation/)
 
-   - Add your space ID to `contentful/import-demo-config.json` and `contentful/import-model-only-config.json`
```

Comment:
I missed adding it to `.contentful/import-model-only-config.json` and thought breaking this instruction into two lines makes it more clear that two actions are required here

Reply:
@thinkybeast and I paired on making this whole part of setup better using a custom script, so these changes might be moot